components/Card/src/css.mjs

```diff
@@ -0,0 +1,255 @@
+  }
+  
+  .denhaag-case-card .denhaag-card__background::after {
+    position: absolute;
+    top: -20px;
+    left: 55%;
+    display: block;
+    width: 10%;
+    height: 22px;
+    content: "";
+    transform: skewX(30deg);
+    background: var(--denhaag-card-case-primary-background-color);
+  }
+  
+  .denhaag-case-card .denhaag-card__arrow-icon {
+    color: var(--denhaag-card-case-arrow-color);
+  }
+  
+  .denhaag-case-card.denhaag-card--hover .denhaag-card__arrow-icon,
+  .denhaag-case-card.denhaag-card--focus .denhaag-card__arrow-icon,
+  .denhaag-case-card:hover .denhaag-card__arrow-icon,
+  .denhaag-case-card:focus-within .denhaag-card__arrow-icon {
+    color: var(--denhaag-card-case-arrow-active-color);
+  }
+  
+  .denhaag-case-card .denhaag-card__title {
+    --utrecht-paragraph-font-weight: var(--denhaag-card-title-font-weight);
+    --utrecht-paragraph-line-height: var(--denhaag-card-title-line-height);
+  
+    color: var(--denhaag-card-case-title-color);
+    padding-block-end: var(--denhaag-card-title-padding-block);
+    padding-block-start: var(--denhaag-card-title-padding-block);
+    padding-inline-end: var(--denhaag-card-title-padding-inline);
+    padding-inline-start: var(--denhaag-card-title-padding-inline);
+  }
+  
+  .denhaag-case-card .denhaag-card__subtitle,
+  .denhaag-case-card .denhaag-card__date-wrapper {
+    --utrecht-paragraph-line-height: var(--denhaag-card-subtitle-line-height);
+  
+    color: var(--denhaag-card-case-color);
+    padding-block-start: var(--denhaag-card-subtitle-padding);
+    padding-block-end: var(--denhaag-card-subtitle-padding);
+    padding-inline-start: var(--denhaag-card-subtitle-padding);
+    padding-inline-end: var(--denhaag-card-subtitle-padding);
+  }
+  
+  .denhaag-case-card.denhaag-card--hover::before,
+  .denhaag-case-card.denhaag-card--focus::before,
+  .denhaag-case-card:hover::before,
+  .denhaag-case-card:focus-within::before {
+    top: 12px;
+    transform: rotate(-2.45deg);
+    transform-origin: right;
+  }
+  
+  .denhaag-case-card.denhaag-card--focus .denhaag-card__background,
+  .denhaag-case-card.denhaag-card--hover .denhaag-card__background,
+  .denhaag-case-card:hover .denhaag-card__background,
+  .denhaag-case-card:focus-within .denhaag-card__background {
+    transform: rotateX(-20deg);
+  }
+  
+  .denhaag-case-card--archived {
+    --denhaag-card-case-primary-background-color: var(--denhaag-card-case-archived-primary-background-color);
+    --denhaag-card-case-secondary-background-color: var(--denhaag-card-case-archived-secondary-background-color);
+    --denhaag-card-case-color: var(--denhaag-card-case-archived-color);
+    --denhaag-card-case-title-color: var(--denhaag-card-case-archived-title-color);
+    --denhaag-card-case-arrow-color: var(--denhaag-card-case-archived-arrow-color);
+    --denhaag-card-case-arrow-active-color: var(--denhaag-card-case-archived-arrow-color);
+  }
+  `;
```

Comment:
Deze zou `.mjs` file met de CSS erin zou nog automatisch gebuild moeten worden, dat is wat Utrecht doet:

`build-css-package`: https://github.com/nl-design-system/utrecht/blob/main/components/code/package.json#L15

en dan
https://github.com/nl-design-system/utrecht/blob/main/packages/build-utils-css/src/index.mjs#L132-L139

Reply:
Ik heb dit eerst opgelost met een rollup uitbreiding. Dat leek het gewenste effect te hebben (in `dist/mjs` kwam een stringified css default export terecht in `css.mjs`. Daarna nog de custom `emitCssDts()` geschreven maar typescript bleef klagen. Moeten we nog een keer naar kijken om dat netjes te doen.

```js
{
    input: 'src/index.scss',
    output: {
      dir: './dist/mjs',
      sourcemap: false,
      entryFileNames: 'css.mjs',
      format: 'esm',
      compact: true,
    },
    plugins: [
      postcss({
        extensions: ['.css', '.scss'],
        extract: false,
        inject: false,
      }),
      emitCssDts(),
    ],
  },
```